recipes/reduce/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "reduce" %}
+{% set version = "4.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rlabduke/reduce/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 62e61cce221fff76b5834031302d91fe703a19945a42e16620d4fb860604daf4
+
+build:
+  skip: True  # [win]
+  number: 0
+  script_env:
+    - CMAKE_PREFIX_PATH=${PREFIX}
+    - CMAKE_INSTALL_PREFIX=${PREFIX}
+
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ stdlib("c") }}
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python =3.*
+    - libboost-python-devel
+  run:
+    - python =3.*
```

Please let this expand the matrix 
```suggestion
    - python
    - libboost-python-devel
  run:
    - python
```

---

The packed tool specifies it needs at least python version 3.

---

As this is a arch-specific package, this will be built anyways per minor Python version. As long as this pin is here, it will not and result in a broken package.

---

The correct way to specify python minimum version is to use [the skip directive in the build section](https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#skipping-builds) along with preprocessing